src/components/Message.tsx

```diff
@@ -28,14 +29,32 @@ const Message: FC<Props> = props => {
   )
   const creationDate = dayjs(props.message.createdAt).format('LLLL')
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const renderLink = ({ attributes, content }: { attributes: any; content: string }) => {
+    const { href } = attributes
+    if (content.startsWith('http')) {
```

Comment:
This condition can go, no? Just always do the replace, and if the regex doesn't match it just does nothing.